docs/design/virtual-addresses.md

```diff
@@ -0,0 +1,123 @@
+# Virtual Addresses
+
+## Purpose
+Introduce the concept of virtual addresses in order to bridge the conceptual gap between Hedera's and Ethereum's view of accounts.
+Virtual addresses will resolve the issue of account EVM compatibility and identification and will be responsible for EVM address related logic.
+
+## Goals
+- Add list of evmAddress values known as “Virtual Address” to Hedera accounts which govern the address the EVM observes for a given account transaction
+- Hedera Accounts can add, disable and remove virtual address entries as desired
+- The Hedera network will validate ownership by extracting the public key from the signature and comparing the calculated public address to the evmAddress passed in on addition of the virtual address and will maintain an `evmAddress -> accountId` map thereafter
+- Contract accounts may utilize the `evmAddress` to store their address in accordance with `CREATE` and `CREATE2` EVM operations
+- Restoring HIP 32’s consistency so that account aliases will have only key based values
```

Comment:
What's this about?  More explanation?  (Perhaps in a separate "comments/notes" section elsewhere in this doc?  Or perhaps a pointer to some other doc, maybe a HIP?)

Reply:
Added some explanation about what is meant by HIP 32 consistency.